src/Device.ts

```diff
@@ -331,6 +334,16 @@ export class Device
 		return this._sctpCapabilities!;
 	}
 
+	/**
+	 * Observer.
+	 *
+	 * @emits newtransport - (transport: Transport)
```

Comment:
There is no  `Transport` class but `SendTransport` and `RecvTransport `.

Reply:
correct

Reply:
Forget about this, my fault.